docs/manual_construction.md

```diff
@@ -0,0 +1,423 @@
+root@gateway1:/# ovs-vsctl add-port br-tun vxlan238 -- set interface vxlan238 type=vxlan options:remote_ip=172.16.62.238 options:key=flow
+```
+
+
+-   **gateway node2 (172.16.62.232)**
+```
+root@gateway2:/# ovs-vsctl add-br br-tun
+root@gateway2:/# ovs-vsctl add-port br-tun vxlan237 -- set interface vxlan237 type=vxlan options:remote_ip=172.16.62.237 options:key=flow
+root@gateway2:/# ovs-vsctl add-port br-tun vxlan238 -- set interface vxlan238 type=vxlan options:remote_ip=172.16.62.238 options:key=flow
+```
+
+-   **gateway node3 (172.16.62.233)**
+```
+root@gateway3:/# ovs-vsctl add-br br-tun
+root@gateway3:/# ovs-vsctl add-port br-tun vxlan237 -- set interface vxlan237 type=vxlan options:remote_ip=172.16.62.237 options:key=flow
+root@gateway3:/# ovs-vsctl add-port br-tun vxlan238 -- set interface vxlan238 type=vxlan options:remote_ip=172.16.62.238 options:key=flow
+```
+
+-   **gateway node4 (172.16.62.234)**
+```
+root@gateway4:/# ovs-vsctl add-br br-tun
+root@gateway4:/# ovs-vsctl add-port br-tun vxlan237 -- set interface vxlan237 type=vxlan options:remote_ip=172.16.62.237 options:key=flow
+root@gateway4:/# ovs-vsctl add-port br-tun vxlan238 -- set interface vxlan238 type=vxlan options:remote_ip=172.16.62.238 options:key=flow
+```
+
+
+
+### 3. OVS and flow table settings for computer nodes (Gateway forwarding)
+#### 3.1 computer node1 (172.16.62.237)
+
+-   **1. Add br-int and br-tun, and create VXLAN tunnels connected to each gateway node.**
+```
+root@computer7:/# ovs-vsctl add-br br-int
+root@computer7:/# ovs-vsctl add-port br-int patch-tun -- set interface patch-tun type=patch options:peer=patch-int
+root@computer7:/# ovs-vsctl add-br br-tun
+root@computer7:/# ovs-vsctl add-port br-tun patch-int -- set interface patch-int type=patch options:peer=patch-tun
+
+root@computer7:/# ovs-vsctl add-port br-tun vxlan231 -- set interface vxlan231 type=vxlan options:remote_ip=172.16.62.231 options:key=flow
+root@computer7:/# ovs-vsctl add-port br-tun vxlan232 -- set interface vxlan232 type=vxlan options:remote_ip=172.16.62.232 options:key=flow
+root@computer7:/# ovs-vsctl add-port br-tun vxlan233 -- set interface vxlan233 type=vxlan options:remote_ip=172.16.62.233 options:key=flow
+root@computer7:/# ovs-vsctl add-port br-tun vxlan234 -- set interface vxlan234 type=vxlan options:remote_ip=172.16.62.234 options:key=flow
+```
+
+-   **2. Start two containers.
+Use ovs-docker to add a nic for the container and hang it to the br-int.
+Set different vlan_id (100 and 200).**
+```
+root@computer7:/# docker run -itd --name docker71 --net none ubuntu /bin/bash
+root@computer7:/# ovs-docker add-port br-int eth0 docker71 --ipaddress=192.168.1.71/24 --mtu=1450
+root@computer7:/# ovs-vsctl show
+root@computer7:/# ovs-vsctl set port 3cf61d844bb54_l tag=100
+
+root@computer7:/# docker run -itd --name docker72 --net none ubuntu /bin/bash
+root@computer7:/# ovs-docker add-port br-int eth0 docker72 --ipaddress=192.168.2.72/24 --mtu=1450

```

Comment:
I am curious on why you need to set the MTU to 1450? Below is the command I used in the test code aca_test_ovs_l3.cpp: 

  cmd_string = "ovs-docker add-port br-int eth0 con3 --ipaddress=" + vip_address_3 +
               "/24 --gateway=" + subnet1_gw_ip + " --macaddress=" + vmac_address_3;

You may want to set the gateway address for the later usage (L3 routing).